src/views/HomePageView/Hero/Hero.jsx

```diff
@@ -27,16 +26,24 @@ import Center from '../../../components/LayoutSections/Center';
 
 // images
 import vectorMapPin from '../../../images/vectorMapPin.svg';
+import sticker2024 from '../../../images/stickers/2024.svg';
+import plasticWrap from '../../../images/textures/plasticWrap.png';
 
 const Hero = () => (
   <StyledHero>
     <Center maxWidth='var(--max-container-width)' gutters='12px' withText>
-      <h2 css={billboardStyle}>
-        28 au 30&nbsp;&nbsp;
+      <HeroTitle css={billboardStyle}>
+        28
+        <br />
+        au 30
         <br />
-        &nbsp;&nbsp;mai&nbsp;
-        <span css={billboardAltStyle}>2024</span>
-      </h2>
+        mai
+        <img src={sticker2024} alt='2024' />
+      </HeroTitle>
+
+      <TextureWrapper>
+        <PlasticTexture src={plasticWrap} alt='' />
```

Comment:
Utiliser **GatsbyImage** plutôt que l'élément HTML img par défaut. Ça activera le lazy loading et ça optimisera un peu au first load de la page. À appliquer pour tous les autres images du même genre.

```suggestion
const Hero = () => {
  const data = useStaticQuery(
    graphql`
      query {
        plasticTexture: file(relativePath: { eq: "textures/plasticWrap.png" }) {
          childImageSharp {
            fixed(width: 600) {
              ...GatsbyImageSharpFixed
            }
          }
        }
      }
    `
  );
  ...
  
  <TextureWrapper>
    <PlasticTexture
        fixed={data.plasticTexture?.childImageSharp?.fixed}
        alt=''
        role='presentation'
     />
  </TextureWrapper>
```